docs/grammar/SolidityLexer.g4

```diff
@@ -47,6 +47,7 @@ FixedBytes:
 	'bytes25' | 'bytes26' | 'bytes27' | 'bytes28' | 'bytes29' | 'bytes30' | 'bytes31' | 'bytes32';
 For: 'for';
 Function: 'function';
+Global: 'global'; // not a real keyword
```

Comment:
This needs to be added to ``identifier`` in the parser part now, like ``From``, ``Error`` and ``Revert``, doesn't it :-)?
We should probably add a test case for using ``global`` as identifier, that would need this :-).
And an issue for making it a keyword in 0.9... although I'm not entirely sure if ``export using`` wasn't actually nicer :-).

Reply:
But I guess this also depends on https://github.com/ethereum/solidity/issues/6482, resp. if we want anything like that... if so, it may be nice to work towards syntax that can be reused...